reana_workflow_controller/workflow_run_manager.py

```diff
@@ -169,6 +149,7 @@ def _workflow_engine_command(self, overwrite_input_parameters=None,
                 ['command'].format(
                     id=self.workflow.id_,
                     workspace=self.workflow.workspace_path,
+                    # XXX: get_specification will be empty
```

Comment:
What's the problem here?

Reply:
Leftover 👍, I had this to remind myself that we may encounter problems in the workflow engines due to `workflow_json` be empty. We might want to get rid of `--workflow-json` at some point as [it's already the case for yadage](https://github.com/reanahub/reana-workflow-engine-yadage/pull/150/files#diff-df867f6b9c0032cabaddb00ae9656658L46-L48).